concordia/views.py

```diff
@@ -1,20 +1,34 @@
 import csv
 import os
 import sys
+import shutil
+from django.http import HttpResponse
 from logging import getLogger
-
 import requests
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.contrib.auth.models import User
+from django.views.generic import TemplateView
+from registration.backends.simple.views import RegistrationView
 from django.core.paginator import Paginator
 from django.db.models import Count, Q, Sum
 from django.http import HttpResponse
 from django.shortcuts import get_object_or_404, redirect, render
 from django.urls import reverse
-from django.views.generic import TemplateView
-from registration.backends.simple.views import RegistrationView
+from django.contrib.auth.models import User
+from django.shortcuts import get_object_or_404
+from django.db import transaction
+from django.db.models import Q
+from django.db.models import Count, Sum
+
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
These lines should have been removed in merge. We use Django settings now (lines 25-31)